ingest/user_de.py

```diff
@@ -0,0 +1,178 @@
+import pandas as pd
+import numpy as np
+import sys
+
+#input file name as CLI
+#python ingest/validate_de.py /Users/mvelyuns/scp-ingest-pipeline/ingest/pairwise.csv
+
+if len(sys.argv) > 1:
+    file_path = sys.argv[1]
+else:
+    file_path = "/Users/mvelyuns/scp-ingest-pipeline/ingest/pairwise.csv"   
```

Comment:
This really should just be a relative path of `ingest/pairwise.csv` but accepting the argument above fixes this.